osquery/tables/system/windows/video_info.cpp

```diff
@@ -25,12 +25,13 @@ QueryData genVideoInfo(QueryContext& context) {
   Row r;
   QueryData results;
 
-  const WmiRequest wmiSystemReq("SELECT * FROM Win32_VideoController");
-  const std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
-  if (wmiResults.empty()) {
+  const auto wmiSystemReq =
+      WmiRequest::CreateWmiRequest("SELECT * FROM Win32_VideoController");
+  if (wmiSystemReq || wmiSystemReq->results().empty()) {
```

Comment:
Here, did you mean `!wmiSystemReq || wmiSystemReq->results().empty()`?

Reply:
Yes, thank you. Good catch.